Cargo.toml

```diff
@@ -60,6 +61,7 @@ mcr-settlement-client = { path = "protocol-units/settlement/mcr/client" }
 mcr-settlement-manager = { path = "protocol-units/settlement/mcr/manager" }
 ## types
 movement-types = { path = "util/movement-types" }
+maptos-rest = { path = "protocol-units/maptos-rest" }
```

Comment:
Maybe add a separate "section" for this with comments, and sort it somewhere between mempool and sequencing.